scaife_viewer/templates/search.html

```diff
@@ -41,10 +42,10 @@ <h3>Search Guide</h3>
       <div class="row">
         <div class="col-sm-3">
           {% if page.object_list.filtered_text_groups %}
-            <h5>Text Groups</h5>
+            <h5>Text Groups{% if request.GET.tg %} <a href="?{% query tg='' %}"><small>clear</small></a>{% endif %}</h5>
```

Comment:
you can just say `{% query tg %]` to delete (rather than set it to empty string

Reply:
(I think!)

Reply:
(actually I guess it doesn't really matter as has the same effect ultimately)

Reply:
Hmm, I don't know how I feel about that honestly. It doesn't feel right to me and didn't even occur to me. Python function call and DTL semantics seem to be working oddly together.

Reply:
Personally I think the best API for this would be to follow JS ES6 object definition combined with a small tweak, for example:

```
{% query tg %} == tg={{ tg }}
{% query -tg %} == remove tg (effectively calling with tg=None)
... current kwarg behavior ...
```

Reply:
yeah, I forgot I delete it if it's falsey anyway so setting to '' gets rid of it. I was thinking of an earlier implementation where I used the *args as well as *kwargs for this purpose

Reply:
Ah! Gotcha. I wouldn't mind implementing my version above at some point if you agree. It does go against normal DTL conventions a little, but the trade-off is much better in composing the query string.